### PR TITLE
Add Icons8

### PR DESCRIPTION
**Pull request by huuphongnguyen**
icons8

**Issue:** #5911 
**Alexa rank:** [5,803](https://www.alexa.com/siteinfo/icons8.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- I got SVG code from View Code of [Icons8.com](https://icons8.com/)
- I used Sketchapp to adjust and export
- I used https://jakearchibald.github.io/svgomg/ to optimize
- I used Inscape to compare with source icons to make sure it fit.

### CODE REVIEW

**Comment by huuphongnguyen** [comment]
> Found off here at #5911.

Yes. I got mention the issue. I cannot link Pull Request to Issue, so wait for maintainer

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4112,6 +4112,11 @@
             "hex": "16A5F3",
             "source": "https://geticonjar.com/"
         },
+        {
+            "title": "Icons8",
+            "hex": "1FB041",
```
Comment (service-paradis) [inline _data/simple-icons.json:4117]: From the website stylesheet, I found `#1FB141`. What do you think?

Comment (service-paradis) [inline _data/simple-icons.json:4117]: From the website stylesheet, I found `#1FB141`. What do you think?

File: icons/icons8.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Icons8</title><path d="M12 0H0v24h12zM18 12c3.3137 0 6-2.6863 6-6s-2.6863-6-6-6-6 2.6863-6 6 2.6863 6 6 6zM18 24c3.3137 0 6-2.6863 6-6s-2.6863-6-6-6-6 2.6863-6 6 2.6863 6 6 6z"/></svg>
```
Comment (service-paradis) [inline icons/icons8.svg:1]: SVG is spot on 💯 , thank you @huuphongnguyen for your contribution!
Once we resolve the color, this can be merged!

**Review by service-paradis — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (huuphongnguyen) [reply _data/simple-icons.json:4117]: Yes. I view code and see it is #1FB141. It's weird when I pick it with [Roy](https://www.useroy.com/), it got #1FB041. I think we should use #1FB141 as source code. I will update it.

**Review by service-paradis — APPROVED** [review]

**Pull request merged**